### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by aniketsingh13**
Screenshot (92)

### CODE REVIEW

**Review by shubhamkumar648 — COMMENTED** [review]
other looks good , see my comment for change

**Review by shubhamkumar648 — COMMENTED** [review]
remove space of line number 70 and look into line 46 class name is empty

**Comment by aniketsingh13** [comment]
> remove space of line number 70 and look into line 46 class name is empty

done with changes

**Review by shubhamkumar648 — APPROVED** [review]
now code looks clean to me  you can merge

File: Components/modal.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PlayX-modal</title>
+    <link rel="stylesheet" href="../Styles/modal.css" />
+    <link rel="stylesheet" href="../Styles/main.css" />
+    <script src="https://kit.fontawesome.com/fa50d5d333.js" crossorigin="anonymous"></script>
```
Comment (Naveenchandar) [inline Components/modal.html]: why script is used for font awesome ?. Font awesome can be imported using link.

Comment (aniketsingh13) [reply Components/modal.html]: don't know how to do using only links. but sure  read about it and do the changes

Comment (Naveenchandar) [reply Components/modal.html]: Please go through this link and get the correct version of whatever you've used in this project.
https://cdnjs.com/libraries/font-awesome

Comment (aniketsingh13) [reply Components/modal.html]: thanks will do that

**Pull request merged**